package.json

```diff
@@ -11,13 +11,23 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
     "css-loader": "^6.8.1",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "hint": "^7.1.10",
     "html-webpack-plugin": "^5.5.3",
     "style-loader": "^3.3.3",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
     "webpack": "^5.38.1",
     "webpack-cli": "^4.7.2"
```

Comment:
- [ ] Adding  `webpack-dev-server": "^4.11.1` as a dependency will aid serving the app locally, kindly fix that too.
<img width="183" alt="image" src="https://github.com/BilalLiaquat7/webpack/assets/41163571/f072508d-37d0-44ca-8942-7cd8f71914fa">